plugins/modules/docker_plugin.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/python
+# coding: utf-8
+#
+# Copyright: (c) 2021 Red Hat | Ansible Sakar Mehra<@sakarmehra100@gmail.com | @sakar97>
```

Comment:
Since you used https://github.com/ansible/ansible/pull/52643 as a basis, you need to include it's copyright and author:
```suggestion
# Copyright: (c) 2021 Red Hat | Ansible Sakar Mehra<@sakarmehra100@gmail.com | @sakar97>
# Copyright: (c) 2019, Vladimir Porshkevich (@porshkevich) <neosonic@mail.ru>
```